kalite/caching/__init__.py

```diff
@@ -65,7 +68,11 @@ def invalidate_all_caches():
     call in here.
     """
     invalidate_inmemory_caches()
-    initialize_content_caches(force=True)
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP:
```

Comment:
Okay, can you explain as much in a comment there? It is perhaps due to my unfamiliarity with this code, but I did not make the connection between deleting these files and not reloading the cache.